Completed merge of mrtk and layers branches into master

grabermtw opened the pull request:
Conflicts between the three branches were resolved. `master` should now have all functionality that `layers` and `mrtk` previously had.

lim-at-infinity committed:
Transitioned from SteamVR to Oculus

lim-at-infinity committed:
Saving files before refreshing line endings

lim-at-infinity committed:
Converted to MRTK and made AR scene

lim-at-infinity committed:
Updated .gitignore

lim-at-infinity committed:
Created initial User Interface

- User is presented panels that can change the settings of a GIBS object

lim-at-infinity committed:
Start Screen Complete

- Currently allows main menu to different scenes
- Utilizes mainmenu.cs & returntomainmenu.cs via Unity SceneManager

lim-at-infinity committed:
User Interface Completed + Added Start Menu Scene

lim-at-infinity committed:
Update .gitignore

grabermtw committed:
Added Coordinates.cs

Coordinates.cs contains a static class Coordinates with static methods allowing for conversion between Unity XYZ world coordinates and latitude and longitude and back.

lim-at-infinity committed:
Created airsFile and airsParser files

lim-at-infinity committed:
Merge pull request #3 from grabermtw/master

Added Coordinates.cs

lim-at-infinity committed:
Updated AIRS files

lim-at-infinity committed:
Added CSV parser to read AIRS data

lim-at-infinity committed:
Fixed variable datatype in airFile.cs

grabermtw committed:
Added DataCurtain.cs, fixed an edge case in Coordinates.cs

DataCurtain.cs generates a data curtain for the sphere it is attached to when Generate() is called.

grabermtw committed:
Removed Debug.Log from DataCurtain.cs

lim-at-infinity committed:
Merge pull request #4 from grabermtw/master

Added DataCurtain.cs

lim-at-infinity committed:
Fixed parser file names, converted airsParser to general CSVParser

grabermtw committed:
Added SphereLinePlot.cs, which can be used to plot data such as that from ICESat-2 as a line plot on the side of a sphere.

Also improved DataCurtain.cs by adding a public method allowing the curtains to be removed via another script.

lim-at-infinity committed:
Merge pull request #5 from grabermtw/master

Added SphereLinePlot.cs

lim-at-infinity committed:
Passed over parsed airsData object to Coordinates.cs, adjusted filepath and parsing handling in CSVParser, added data to assets

lim-at-infinity committed:
Re-arranged some files, wrote basic json reader

To-do:
- Need to make it so that json reader can serialize the 5 different outputs
- Connect parsed data with rendering scripts

lim-at-infinity committed:
Moved data location, added AirsRender.cs

AirsRender.cs instantiates the data points.

Issues:
GameObjects do not currently render on the GIBSObject

lim-at-infinity committed:
Update AirsRender.cs

grabermtw committed:
Added ICESat2Getter.cs

ICESat2Getter.cs gets ICESat-2 data from OpenAltimetry. Also fixed an overloading error in SphereLinePlot.cs.

grabermtw committed:
Added ICESat2Getter.cs, fixed overload error in SphereLinePlot.cs

ICESat2Getter.cs gets ICESat-2 data from the OpenAltimetry API.

lim-at-infinity committed:
Merge pull request #6 from grabermtw/mrtk

Added ICESat2Getter.cs, fixed overload error in SphereLinePlot.cs

lim-at-infinity committed:
Adjustments to AIRS Visualizer

lim-at-infinity committed:
Merge branch 'mrtk' of https://github.com/lim-at-infinity/gibs-unity-examples into mrtk

lim-at-infinity committed:
Cleaned up AirsRender.cs code

lim-at-infinity committed:
Added geojson parser scripts

- GeojsonData.cs
- GeojsonHelper.cs

lim-at-infinity committed:
Adjusted AIRS Color Scale, finished Geojson Parser

- Need to handle texture for GeojsonInstantiator

lim-at-infinity committed:
Casted Resources.Load() as <Texture>

lim-at-infinity committed:
Fixed CalipsoReader() call runtime error

lim-at-infinity committed:
Preparing code before changing data vriables from float to double

In order to make sure the calipso data is set to the correct precision, the code is being rewritten to use doubles instead of floats. This is a version right before the conversion.

lim-at-infinity committed:
Geojson now deserializes correctly

lim-at-infinity committed:
Calipso Visualization Completed

lim-at-infinity committed:
ICESat2 Data Loader

- LinePlot is not rendering correctly, may need to look at LinePlot.cs

lim-at-infinity committed:
ICESat2 visualization done

lim-at-infinity committed:
Safety Push before MRTK upgrade

lim-at-infinity committed:
Upgraded to MRTK 2.4.0

grabermtw committed:
Added ability for SphereLinePlot to plot discrete points

Also fixed bug with ICESat2Loader

lim-at-infinity committed:
Merge pull request #7 from grabermtw/mrtk

Added ability for SphereLinePlot to plot discrete points

lim-at-infinity committed:
Final Commit Winter 2020

lim-at-infinity committed:
Final Commit Summer 2020

SoLucyCreates committed:
Added dropdown menu to select layers

grabermtw committed:
Merge pull request #1 from lim-at-infinity/mrtk

Mrtk

grabermtw committed:
Merge pull request #2 from nasa-gibs/mrtk

Pull Final Commit Summer 2020

grabermtw committed:
Merge branch 'mrtk'

grabermtw committed:
Removed unnecessary duplicate of ICESat2Getter.cs

LaurenGold committed:
Merge branch 'master' into layers

LaurenGold committed:
Create wms1.unity

LaurenGold committed:
Resolving conflicts

- Missing scripts/dependencies
- added mrtk to resolve most of it
- commented out line 105 of MixedRealitySceneContent.cs to resolve error
- added missing script on wms globe (nearinteractiongrabber.cs)

LaurenGold committed:
Update .gitignore

LaurenGold committed:
Merge branch 'layers'

jtroberts commented:
Is the MSBuild/ directory and MSBuildForUnity.Common.props file necessary? There are a few files in there containing user specific paths that shouldn't be included in an open source repository. Otherwise looks good (although I didn't actually review all 4500+ files).

grabermtw committed:
Delete MSBuildForUnity.Common.props

this was supposed to have been gitignore'd

grabermtw committed:
Delete MSBuild directory

this directory was supposed to have been gitignore'd

LaurenGold commented:
I see what you're talking about, and it seems useless considering the paths
I'm seeing thus far are from Angel's computer. However, I'm not entirely
sure about its necessity. I can look into it.

On Fri, Jun 25, 2021 at 7:57 PM Joe T. Roberts ***@***.***>
wrote:

> Is the MSBuild/ directory necessary? There are a few files in there
> containing user specific paths that shouldn't be included in an open source
> repository. Otherwise looks good (although I didn't actually review all
> 4500+ files).
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/nasa-gibs/gibs-unity-examples/pull/8*issuecomment-868899176__;Iw!!IKRxdwAv5BmarQ!K35-QbifWMbP9KVoBA1S9Phzd1FN4-PVB6Ot-BqIXLOR9MJBGygUlDn0RrRyGA$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AJRRDIO46CGWM4GOOHUCLWLTUUQYJANCNFSM47KOKQSA__;!!IKRxdwAv5BmarQ!K35-QbifWMbP9KVoBA1S9Phzd1FN4-PVB6Ot-BqIXLOR9MJBGygUlDm277Bubg$>
> .
>


-- 
Thank you,
Lauren Gold (she/her)

*Meteor Studio* | Arizona State University
Graduate Student Researcher & Developer
website: Lauren-Gold.com
phone: 972-533-1107

grabermtw committed:
Update .gitignore

fixed `MSBuildForUnity.Common.props`

grabermtw commented:
> I see what you're talking about, and it seems useless considering the paths I'm seeing thus far are from Angel's computer. However, I'm not entirely sure about its necessity. I can look into it.
> […](#)
> On Fri, Jun 25, 2021 at 7:57 PM Joe T. Roberts ***@***.***> wrote: Is the MSBuild/ directory necessary? There are a few files in there containing user specific paths that shouldn't be included in an open source repository. Otherwise looks good (although I didn't actually review all 4500+ files).

Joe is right, those files had been listed in the .gitignore but for some reason had not been ignored. One of them had an unnecessary `.` at the start of the filename that probably messed with things. I removed those files and updated the .gitignore.

LaurenGold commented:
Ohhhh I see. Thanks Matt

On Mon, Jun 28, 2021 at 8:39 AM Matt Graber ***@***.***>
wrote:

> I see what you're talking about, and it seems useless considering the
> paths I'm seeing thus far are from Angel's computer. However, I'm not
> entirely sure about its necessity. I can look into it.
> … <#m_-9021414104037926226_>
> On Fri, Jun 25, 2021 at 7:57 PM Joe T. Roberts *@*.***> wrote: Is the
> MSBuild/ directory necessary? There are a few files in there containing
> user specific paths that shouldn't be included in an open source
> repository. Otherwise looks good (although I didn't actually review all
> 4500+ files). — You are receiving this because you were assigned. Reply to
> this email directly, view it on GitHub <https://urldefense.com/v3/
> *https://github.com/nasa-gibs/gibs-unity-examples/pull/8*issuecomment-868899176*
> ;Iw!!IKRxdwAv5BmarQ!K35-QbifWMbP9KVoBA1S9Phzd1FN4-PVB6Ot-BqIXLOR9MJBGygUlDn0RrRyGA$
> <https://urldefense.com/v3/__https://github.com/nasa-gibs/gibs-unity-examples/pull/8*issuecomment-868899176__;Iw!!IKRxdwAv5BmarQ!K35-QbifWMbP9KVoBA1S9Phzd1FN4-PVB6Ot-BqIXLOR9MJBGygUlDn0RrRyGA%24>>,
> or unsubscribe <https://urldefense.com/v3/
> *https://github.com/notifications/unsubscribe-auth/AJRRDIO46CGWM4GOOHUCLWLTUUQYJANCNFSM47KOKQSA*
> ;!!IKRxdwAv5BmarQ!K35-QbifWMbP9KVoBA1S9Phzd1FN4-PVB6Ot-BqIXLOR9MJBGygUlDm277Bubg$
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AJRRDIO46CGWM4GOOHUCLWLTUUQYJANCNFSM47KOKQSA__;!!IKRxdwAv5BmarQ!K35-QbifWMbP9KVoBA1S9Phzd1FN4-PVB6Ot-BqIXLOR9MJBGygUlDm277Bubg%24>>
> .
> -- Thank you, Lauren Gold (she/her) *Meteor Studio* | Arizona State
> University Graduate Student Researcher & Developer website: Lauren-Gold.com
> phone: 972-533-1107
>
> Joe is right, those files had been listed in the .gitignore but for some
> reason had not been ignored. One of them had an unnecessary . at the
> start of the filename that probably messed with things. I removed those
> files and updated the .gitignore.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/nasa-gibs/gibs-unity-examples/pull/8*issuecomment-869693579__;Iw!!IKRxdwAv5BmarQ!Nl3CdxZdQ1Au4gWvWuDyokuOJLY7cVJcXiQByW0esMPX-4nDYTDV0UyEHCDJNQ$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AJRRDINPDCAUAX63CICHDXLTVB3PTANCNFSM47KOKQSA__;!!IKRxdwAv5BmarQ!Nl3CdxZdQ1Au4gWvWuDyokuOJLY7cVJcXiQByW0esMPX-4nDYTDV0UwTviV-Iw$>
> .
>


-- 
Thank you,
Lauren Gold (she/her)

*Meteor Studio* | Arizona State University
Graduate Student Researcher & Developer
website: Lauren-Gold.com
phone: 972-533-1107
